babel.config.js

```diff
@@ -1,8 +1,15 @@
-module.exports = {
-    presets: [
-        require('metro-react-native-babel-preset'),
-    ],
-    plugins: [
-        '@babel/plugin-proposal-class-properties',
-    ],
+module.exports = (api) => {
+    const isWeb = api.caller(caller => caller && caller.target === 'web');
+    if (isWeb) {
+        return {
+            // Default browser list is a reasonable preset covering a wide list of non-dead browsers
+            // https://github.com/browserslist/browserslist#best-practices
+            targets: 'defaults',
+            presets: ['@babel/preset-env', '@babel/preset-react'],
+        };
+    }
```

Comment:
We do something similar in App, where the webpack build is using preset env and preset react